### PR TITLE
feat: sealos inspect

### LINKED ISSUE #1160: Feature:  sealos inspect
1. sealos inspect  imageName
2. sealos inspect containerName

### PR DESCRIPTION
**Pull request by gitccl**
- resolve #1160

### CODE REVIEW

File: cmd/sealos/cmd/inspect.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	bb "github.com/labring/sealos/pkg/image/buildah/cluster/buildah"
+	"github.com/labring/sealos/pkg/utils/exec"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+)
+
+var exampleInspect = `
+  sealos inspect imageName
+  sealos inspect containerName
```
Comment (fanux) [inline cmd/sealos/cmd/inspect.go:33]: containerName->clusterName, container is cluster, distinguish with docker container.

Comment (gitccl) [reply cmd/sealos/cmd/inspect.go:33]: Is clusterName specified by the  --name parameter of `sealos run` command?

File: cmd/sealos/cmd/inspect.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.Errorf("container or image name must be specified")
+			}
+			if len(args) > 1 {
+				return errors.Errorf("too many arguments specified")
+			}
+
+			name := args[0]
+			if _, ok := exec.CheckCmdIsExist("buildah"); ok {
```
[the remark this answers is not in the document]
Comment (gitccl) [reply cmd/sealos/cmd/inspect.go:50]: You mean it needn't use buildah binary, we can directly use the the packages of images?

**Comment by gitccl** [comment]
I'm testing it.

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**